prometheusbp/testutil.go

```diff
@@ -0,0 +1,68 @@
+package prometheusbp
+
+import (
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/testutil"
+)
+
+// PrometheusMetricTest stores information about a metric to use for testing.
+type PrometheusMetricTest struct {
+	t           *testing.T
```

Comment:
in general, prefer to use `tb testing.TB` so that this struct can also be used in benchmark tests (although it probably doesn't make sense to use this in benchmark tests)

Reply:
@fishy  thats a good idea. How about for now, leave as it, but if this ends up being used in benchmark tests int he future, we can change it to `tb testing.TB` then. Does that sound ok?

Reply:
that's fine, but this is a trivial change so I don't understand the push back.

Reply:
oops sorry wasn't trying to push back at all. im totally happy to make the change. i think i misunderstood your comment... I didn't know if you wanted me to make the change or not because of this part "(although it probably doesn't make sense to use this in benchmark tests)".  But I don't feel strongly either way and I'm totally happy to change it.